utils.py

```diff
@@ -87,6 +156,20 @@ def add_ee_catalog_image(name, asset_name, visParams, collection_props):
 
     add_or_update_ee_image_layer(image, name)
 
+
 def check_version():
     # check if we have the latest version only once plugin is used, not once it is loaded
     qgis.utils.plugins['ee_plugin'].check_version()
+
+
+def geom_to_geo(geom):
```

Comment:
What does this do?

Reply:
It converts Qgs geometry to geo, used in provider.py